src/src/Palindrom.java

```diff
@@ -0,0 +1,19 @@
+public class Palindrom {
+
```

Comment:
General recommendations:
1. Format your code all the time
2. Do not leave empty lines